code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -738,36 +840,35 @@
+	name = "\improper psychologist's winter coat"
+	desc = "A simple white coat with a brown trim, perfect for its job. Just a tad bit resistant to chemicals."
+	icon_state = "wintercoat_psych"
+	inhand_icon_state = "wintercoat_psych"
+	armor = list("melee" = 5, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 10)
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/medical/psych
 
-/obj/item/clothing/suit/hooded/wintercoat/science
-	name = "science winter coat"
-	desc = "A white winter coat with an outdated atomic model instead of a plastic zipper tab."
-	icon_state = "coatscience"
-	inhand_icon_state = "coatscience"
-	allowed = list(/obj/item/analyzer, /obj/item/stack/medical, /obj/item/dnainjector, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray, /obj/item/healthanalyzer, /obj/item/flashlight/pen, /obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/glass/beaker, /obj/item/reagent_containers/pill, /obj/item/storage/pill_bottle, /obj/item/paper, /obj/item/melee/classic_baton/telescopic, /obj/item/toy, /obj/item/storage/fancy/cigarettes, /obj/item/lighter, /obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman)
-	armor = list(MELEE = 0, BULLET = 0, LASER = 0,ENERGY = 0, BOMB = 10, BIO = 0, RAD = 0, FIRE = 20, ACID = 0)
-	hoodtype = /obj/item/clothing/head/hooded/winterhood/science
-	species_exception = list(/datum/species/golem)
+/obj/item/clothing/head/hooded/winterhood/medical/psych
+	desc = "A brown hood that is also slightly resistant to chemicals, you should probably avoid them while wearing this anyways."
+	icon_state = "winterhood_psych"
+	armor = list("melee" = 5, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 10)
 
-/obj/item/clothing/head/hooded/winterhood/science
-	desc = "A white winter coat hood. This one will keep your brain warm. About as much as the others, really."
-	icon_state = "winterhood_science"
-	armor = list(MELEE = 0, BULLET = 0, LASER = 0,ENERGY = 0, BOMB = 10, BIO = 0, RAD = 0, FIRE = 20, ACID = 0)
+
+// SCIENCE
 
 /obj/item/clothing/suit/hooded/wintercoat/science/rd
 	name = "research director's winter coat"
 	desc = "A thick arctic winter coat with an outdated atomic model instead of a plastic zipper tab. Most in the know are heavily aware that Bohr's model of the atom was outdated by the time of the 1930s when the Heisenbergian and Schrodinger models were generally accepted for true. Nevertheless, we still see its use in anachronism, roleplaying, and, in this case, as a zipper tab. At least it should keep you warm on your ivory pillar."
-	icon_state = "coatrd"
-	inhand_icon_state = "coatrd"
+	icon_state = "wintercoat_rd"
+	inhand_icon_state = "wintercoat_rd"
 	armor = list(MELEE = 0, BULLET = 0, LASER = 0,ENERGY = 0, BOMB = 20, BIO = 0, RAD = 0, FIRE = 30, ACID = 0)
```

Comment:
... why do normal winter coats get 5 points of melee armor, but not some of these subtypes?

I mean, this PR didn't cause that weirdness, but maybe it could change/fix it. 🤷

Reply:
I tried to catch all instences of this weirdness left by the last pr, but this is one of them i missed, ill fix it.